crates/experimental-api/src/gen.rs

```diff
@@ -2162,6 +2185,312 @@ pub mod gen {
         pub state_commitment: Option<Felt>,
     }
 
+    impl GetProofResult {
```

Comment:
Please move out the implementation out of `gen.rs`, it is only for DTOs and relevant validation.

Reply:
Migrated to newly created proof_validation module